modules/local_contacts/modules.php

```diff
@@ -173,3 +229,27 @@ protected function output() {
             $this->trans('Cancel').'" /></div></form></div>';
     }
 }
+
+/**
+ * @subpackage import_local_contacts/output
+ */
+class Hm_Output_import_contacts_form extends Hm_Output_Module {
+    protected function output() {
+
+        $form_class = 'contact_form';
+        $button = '<input class="add_contact_submit" type="submit" name="add_contact" value="'.$this->trans('Add').'" />';
+        $notice = 'Please ensure your CSV header file follows the format: display_name, email_address, phone_number';
```

Comment:
Probably better idea is to provide a sample csv file. You can put a static one in a web-accessible URL. Something as small as:
```
display_name, email_address, phone_number
Thomas Tester, test@example.org, 1234567890
```
and then have a link to download a sample csv file.